test/integration/ResponseValidationTest.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const TestHelper = require('../TestHelper');
+const mockyeah = TestHelper.mockyeah;
+const request = TestHelper.request;
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('Response Validation', () => {
+  it('should validate response option(s) are correct', done => {
+    try {
+      // Use incorrect option 'file'
+      mockyeah.get('/some/service/end/point', { file: './fixtures/some-data.json' });
+    } catch (err) {
+      expect(err.message)
+      .to.equal('Response option(s) invalid');
```

Comment:
Yes good catch, I had previously had a long error message that forced me to line wrap. I'll fix this up!